src/backend/gporca/libnaucrates/src/parser/CParseHandlerMDRelation.cpp

```diff
@@ -299,6 +303,7 @@ CParseHandlerMDRelation::EndElement(const XMLCh *const,	 // element_uri,
 	CMDColumnArray *md_col_array = md_cols_parse_handler->GetMdColArray();
 	CMDIndexInfoArray *md_index_info_array =
 		pphMdlIndexInfo->GetMdIndexInfoArray();
+	GPOS_ASSERT(NULL != pphMdidlTriggers);
```

Comment:
move this assert to the rest of the group.

Reply:
moved